README.md

```diff
@@ -1,71 +1,145 @@
+
 # WasmEdge Docs
 
 This repo contains technical documentation for the [WasmEdge Runtime](https://github.com/WasmEdge/WasmEdge) project. The documentation website built from this repo is published [here](https://wasmedge.org/docs/), check it out!
+This repository contains the technical documentation for the [WasmEdge Runtime](https://github.com/WasmEdge/WasmEdge) project. The documentation website built from this repo is published [here](https://wasmedge.org/docs/). It contains all the necessary information on what WasmEdge Runtime is and how to develop Wasm app, embed WasmEdge into your host applications, and extend/contribute to WasmEdge.
 
+Check it out!
 https://wasmedge.org/docs/
 
 # Setting up the Docs Locally
+## Features
+
+### [Develop](https://wasmedge.org/docs/develop/overview)
+
+It provides information on:
+1. How to get started with WasmEdge
+2. How to develop Wasm apps in various languages (C/C++, Go, Rust, JavaScript)
+3. How to deploy WasmEdge apps in Kubernetes 
+
+### [Embed](https://wasmedge.org/docs/embed/overview)
+
+It provides information on how to embed Wasm functions using various SDKs and some use cases.
+
+### [Extend](https://wasmedge.org/docs/contribute/overview)
+
+It provides insights on how to start contributing to the WasmEdge, Ideas for contribution and Mentoring.
+
+###
+
+## Configuring docs to local machine
 
 To set up the WasmEdge Docs locally, you will need to follow these general steps:
+To set up this repo to your local machine, follow the below steps:
 
 1. Clone the project : Go to the GitHub repository page of the WasmEdge Docs project and click on the "Code" button to get the URL of the repository. Then, open a terminal or command prompt and type the following command:
+1. Clone the repo : Go to the GitHub repository of the WasmEdge Docs project and click on the "Code" button. Copy the https link of the repository. Typethe below command in the terminal or command prompt:
 ```
 git clone https://github.com/WasmEdge/docs.git
+git clone https://github.com/WasmEdge/docs.git/
 ```
 
 1. Install dependencies : Navigate to the project directory in the terminal and run the following command to install the necessary dependencies:
+2. Installing the dependencies : Navigate to the project directory in the terminal and run the following command to install the necessary dependencies:
 ```
 npm install
 ```
 1. Start the development server : Once the dependencies are installed, you can start the development server by running the following command:
+
+3. Start the development server : To start the development server, run the following command:
 ```
 npm start
 ```
 
 This should start the development server on port 3000. You can access the running application by opening a web browser and navigating to http://localhost:3000/docs/.
+You can access the running application by opening a web browser and navigating to http://localhost:3000/docs/.
+
+
+## Contributing to the Docs
```

Comment:
Please remove this part. Since Contributing To The Docs is a new independent file. Thanks.